packages/core/src/shared/components/toolbar/SimpleToolbarButton.tsx

```diff
@@ -29,27 +29,27 @@ export const SimpleToolbarButton = forwardRef(
             secondaryTooltip={props.secondaryTooltip}
           />
         }>
-        <Button
-          ref={ref as any}
-          appearance="subtle"
-          onClick={props.onClick}
-          isSelected={props.isSelected || false}
-          isDisabled={props.isDisabled || false}
-          iconBefore={
-            ButtonIcon && (
-              <ButtonIcon
-                className={
-                  styles.icon +
-                  " " +
-                  (props.isSelected ? styles.isSelected : "") +
-                  " " +
-                  (props.isDisabled ? styles.isDisabled : "")
-                }
-              />
-            )
-          }>
-          {props.children}
-        </Button>
+        {props.children ? (
```

Comment:
when do we use the variant with children?

Reply:
In `HyperlinkBasicMenu.tsx`, it's used for an edit button for existing hyperlinks.

Reply:
Could be changed but this is how it was in the AtlasKit UI.